TheFactorial.py

```diff
@@ -0,0 +1,12 @@
+number = input("Enter a value: ")
+number = int(number)
+
+def factorial(n):
+    if n == 0 or n == 1:
+        return 1
+    result = n * factorial(n - 1)
+
+    return result
+
+
+print("The factorial of 5 is:  " + str(factorial(number)))
```

Comment:
You should print **f"The factorial of {number} is: "** not **"The factorial of 5 is: "** .
When you print **factorial(number)** is not needed to convert it to string.

Reply:
First statement is done. The second one is not ok. It needs to be converted, because when entering from keyboard, the number is a string, then I have to convert it into int for the function to operate with it, and for ptinting, I have typeErrot if I don't reconvert the number into a string.

![image](https://user-images.githubusercontent.com/76015022/142463117-5d605819-fa5e-4326-b2ce-39890af86391.png)